runtime/doc/nvim_terminal_emulator.txt

```diff
@@ -117,8 +117,8 @@ variables:
 - *b:term_title* The settable title of the terminal, typically displayed in
   the window title or tab title of a graphical terminal emulator. Programs
   running in the terminal can set this title via an escape sequence.
-- *b:terminal_job_id* The nvim job ID of the job running in the terminal. See
-  |job-control| for more information.
+- *b:terminal_job_id* The nvim channel ID or the PTY of the terminal emulateor. See
```

Comment:
should be `channel ID of the PTY of the emulatator`, but I guess it could be clearer; the channelrepresents specifically the master end of the PTY, not the PTY itself.